quickstart/src/main/java/com/microsoft/azure/kusto/quickstart/KustoSampleApp.java

```diff
@@ -0,0 +1,461 @@
+                break;
+            case "AppCertificate":
+                csb = createAppCertificateStringBuilder(endpointUrl);
+                break;
+            case "DeviceCode":
+                csb = ConnectionStringBuilder.createWithDeviceCode(endpointUrl);
+                break;
+            default:
+                die(String.format("Authentication mode %s is not supported", authenticationMode));
+        }
+
+        csb.setApplicationNameForTracing("KustoJavaSdkQuickStart");
+        return csb;
+    }
+
+    private static ConnectionStringBuilder createAppCertificateStringBuilder(String endpointUrl) {
+        String appId = System.getenv("APP_ID");
+        String tenantId = System.getenv("APP_TENANT");
+        String privateKeyFilePath = System.getenv("PRIVATE_KEY_PEM_FILE_PATH");
+        String publicCertFilePath = System.getenv("PUBLIC_CERT_FILE_PATH");
+
+        try {
+            PrivateKey privateKey = SecurityUtils.getPrivateKey(publicCertFilePath);
+            X509Certificate x509Certificate = SecurityUtils.getPublicCertificate(privateKeyFilePath);
+            return ConnectionStringBuilder.createWithAadApplicationCertificate(endpointUrl, appId, x509Certificate, privateKey, tenantId);
+        } catch (IOException | GeneralSecurityException e) {
+            die("Couldn't create ConnectionStringBuilder for app certificate authentication", e);
+            return null;
+        }
+    }
+
+    private static void createNewTable(Client kustoClient, String databaseName, String tableName, String tableSchema) {
+        waitForUserToProceed(String.format("Create table '%s.%s' if it doesn't exist", databaseName, tableName));
+        String command = String.format(".create table %s %s", tableName, tableSchema);
+        if (!executeControlCommand(kustoClient, databaseName, command)) {
+            die("Failed to create table or validate it exists");
+        }
+    }
+
+    private static void alterExistingTableToProvidedSchema(Client kustoClient, String databaseName, String tableName, String tableSchema) {
+        waitForUserToProceed(String.format("Alter existing table '%s.%s' to align with the provided schema", databaseName, tableName));
+        String command = String.format(".alter table %s %s", tableName, tableSchema);
+        if (!executeControlCommand(kustoClient, databaseName, command)) {
+            die("Failed to alter table");
+        }
+    }
+
+    private static boolean executeControlCommand(Client kustoClient, String databaseName, String controlCommand) {
+        ClientRequestProperties clientRequestProperties = createClientRequestProperties("SampleApp_ControlCommand");
+        KustoOperationResult result;
+        try {
+            result = kustoClient.execute(databaseName, controlCommand, clientRequestProperties);
+            System.out.printf("Response from executed control command '%s':%n", controlCommand);

```

Comment:
Add comment that printing is only to help demonstrate behind the scenes, and you likely wouldn't